runtime/pom.xml

```diff
@@ -145,16 +145,65 @@
             SpecV1AddressTest.test79, SpecV1AddressTest.test80, SpecV1AddressTest.test81,
             SpecV1AddressTest.test87, SpecV1AddressTest.test88, SpecV1AddressTest.test89,
             SpecV1AddressTest.test90, SpecV1AddressTest.test91,
-            <!-- Need assert trap and active/passive memory initialization to be completed -->
-            SpecV1LinkingTest.test83, SpecV1LinkingTest.test84, SpecV1LinkingTest.test86,
-            SpecV1LinkingTest.test88, SpecV1LinkingTest.test89,
-            <!-- Require assert_trap in module initialization -->
-            SpecV1LinkingTest.test74, SpecV1LinkingTest.test75, SpecV1LinkingTest.test77,
-            <!-- Active/Passive after module failed initialization -->
-            SpecV1LinkingTest.test103, SpecV1LinkingTest.test104, SpecV1LinkingTest.test106,
-            SpecV1LinkingTest.test109, SpecV1LinkingTest.test110
+            <!-- Those tests rely on other tests failures ... -->
+            SpecV1LinkingTest.test83, SpecV1LinkingTest.test86, SpecV1LinkingTest.test94,
+            SpecV1LinkingTest.test97, SpecV1LinkingTest.test115, SpecV1LinkingTest.test116,
+            SpecV1LinkingTest.test118, SpecV1LinkingTest.test121, SpecV1LinkingTest.test122
           </excludedTests>
-          <excludedValidationTests>binary.wast,binary-leb128.wast</excludedValidationTests>
+          <excludedMalformedTests>binary.wast,
+            binary-leb128.wast</excludedMalformedTests>
+          <excludedInvalidTests>block.wast,
```

Comment:
I'm not understanding the difference between malformed and invalid tests, can you please explain?

Reply:
They come straight from the `wast` representation and they are differentiated there, here I'm just preserving the original distinction.

From what I see I can attempt:
- `assert_malformed` -> there is a problem in the binary representation
- `assert_invalid` -> the binary representation is valid but there is a semantic issue e.g. a `block` that should return i32 returns f64 instead